fuzz_test.go

```diff
@@ -0,0 +1,20 @@
+package snappy
+
+import "testing"
+
+func FuzzData(f *testing.F) {
+	f.Fuzz(func(t *testing.T, data []byte) {
+		func() int {
```

Comment:
:thinking: I'm not sure what the inner nested anonymous func is for? I'm not that familiar with the new `f.Fuzz` though...

Reply:
@eapache yeah that's the fuzz target (as opposed to the fuzz test) which defines what arguments to expect from the corpus and testcases:

![image](https://user-images.githubusercontent.com/8060970/211797353-0f92344b-9f94-448d-b119-d2931f74dd68.png)